.test/unit/test.cpp

```diff
@@ -1949,7 +1985,7 @@ TEST_CASE( "Hypercomplex: Polynomial lib test", "[unit]" ) {
             REQUIRE( hA[2] == polynomial3 % 2 );
             REQUIRE( hA[3] == polynomial4 % 2 );
         }
-
+        
```

Comment:
maybe you want to avoid trailing whitespaces

Reply:
Done! Thanks!